### ISSUE TITLE
Replace Firefox Monitor: Get Mozilla VPN banner with 4.99 pricing special banner

### ISSUE THREAD

**Issue by tcinotto** [mozilla/blurts-server#2193]
We are running a new campaign for Mozilla VPN, before we end our introductory pricing and need to replace the banner on Firefox Monitor.

Link to asset: https://drive.google.com/drive/folders/16ghFYfkLYPGm60ZqSacXS-eMCU5ItAXs?usp=sharing

This banner should only go live in US, UK, CA, Singapore, New Zealand, and Malaysia in English only. 

The banner should be dismissable.

UTM parameter: https://www.mozilla.org/products/vpn/?utm_source=firefox-monitor&utm_medium=firefox-monitor&utm_campaign=intro-pricing 

Asset should stay live until July 13th then be removed. I'll file another issue for changing the asset back to the existing asset.

**Comment by tcinotto** [comment]
Issue for removal: https://github.com/mozilla/blurts-server/issues/2194

**Comment by maxxcrawford** [comment]
> This banner should only go live in US, UK, CA, Singapore, New Zealand, and Malaysia in English only.

**Note that this banner is only gated by language only. Any country with English as the primary/preferred language will see this banner.**
